portfolio/src/main/webapp/index.html

```diff
@@ -21,189 +19,184 @@ <h1 class="headline-text">Dagmawi Haile</h1>
-                <div class="info">
-                    <ul class="courses">
-                        <li> 
-                            Created and co-taught a six-week Python bootcamp to a group of fourty students ranging from 6th-11th
-                            grade from the local Roxbury, MA neighborhood and Ethiopian community.
-                        </li>
-                    </ul>
-                </div>
-            </div>
+          <div class="section">
+            <div class="header">
+            <h3>Google</h3>
+            <h4>STEP Intern</h4>
+            <p>Jun 2020 - Sep 2020</p>
+          </div>
+          <div class="info">
+            <ul class="courses">
+              <li> 
+                Completed a series of excercises spanning across HTML, CSS, JavaScript, and Java Servlets
+                which introduced me to web application design and development
+              </li>
+              <li>
+                Used the HTML, CSS, JavaScript, and Java Servlets skills leared in the first portion of the 
+                internship to create am open source web application.
+              </li>
+            </ul>
+          </div>
+        </div>
+        <div class="section">
+          <div class="header">
+            <h3>MIT Media Lab: Viral Communcations</h3>
+            <h4>Undergraduate Research Assistant</h4>
+            <p>Jun 2019 - Aug 2019</p>
+          </div>
+          <div class="info">
+            <ul class="courses">
+              <li> 
+                Devloped the user interface for a bias reducing new streamin app called Enlightened using
+                React Native
+              </li>
+              <li>
+                Used Google Firebase and JavaScript to handle user authentication and create the data structure
+                used for Enlightened
+              </li>
+            </ul>
+          </div>
+        </div>
+        <div class="section">
+          <div class="header">
+            <h3>MIT EESA Python Summer Camp</h3>
+            <h4>Co-Teacher & Organizer</h4>
+            <p>Jun 2019 - Aug 2019</p>
+          </div>
+          <div class="info">
+            <ul class="courses">
+              <li> 
+                Created and co-taught a six-week Python bootcamp to a group of fourty students ranging from 6th-11th
+                grade from the local Roxbury, MA neighborhood and Ethiopian community.
+              </li>
+            </ul>
+          </div>
+        </div>
         </div> 
-        </section> 
+      </div>
+      <div class="right"></div>
     </div>
-     
     <div class="projects">
-    <section>
       <h1 class="titles">Projects</h1>
+      <hr id="proj">
       <div class="container">
         <div class="cards">
-          <a class="card">
-            <span class="card-thumbnail thumb1">
-              <span><h4 class="card-topic">Community</h4></span>
+          <a class="card" href="finalProj.md.html" target="_blank" >
```

Comment:
A little unusual to have .md.html here, some OS's may not like that. Any particular reason for it?

Reply:
It is a link to a final write up for a project I did. It's not hosted anywhere but I can put it on drive if that is better.